spec/services/rdf_fetcher_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe RdfFetcher do
+  describe ".call" do
+    let(:url) { "http://example.com" }
+    let(:graph) { instance_double("RDF::Graph") }
+    let(:result) { RdfFetcher.call(url) }
+
+    before do
+      allow(RDF::Graph).to receive(:load).with(url).and_return(graph)
+    end
+
+    context "when the URL's scheme is http" do
+      let(:url) { "http://example.com" }
+      it "should return the graph" do
+        expect(result).to eq(graph)
+      end
+    end
+
+    context "when the URL's scheme is https" do
+      let(:url) { "https://example.com" }
+      it "should return the graph" do
+        expect(result).to eq(graph)
+      end
+    end
+
+    context "when the URL's scheme is invalid" do
+      let(:url) { "gopher://example.com" }
```

Comment:
Nothing!  It's just a wonderful example of something we probably have no need to support :)

For simplicity, I'm forcing the scheme to http/https.  We can reconsider this if there's a use case, but we have to at least ensure local file paths are disallowed.  (It'll let you put in an absolute path to a local file since it's just a call to `RDF::Graph.load`)